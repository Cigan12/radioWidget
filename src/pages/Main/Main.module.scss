@import '../../static/styles/variables';

.MainPage {
    background-color: $dark-blue-800;
    width: 100%;
    height: 100vh;
    color: $gray;
}
