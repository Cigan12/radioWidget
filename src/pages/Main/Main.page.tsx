import React from 'react';
import { RadioWidget } from '../../components/Widgets/RadioWidget/RadioWidget.component';
import styles from './Main.module.scss';

export const MainPage: React.FC = () => {
    return (
        <div className={styles.MainPage}>
            <RadioWidget />
        </div>
    );
};
