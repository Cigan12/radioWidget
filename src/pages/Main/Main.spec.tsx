import React from 'react';
import { MainPage } from './Main.page';
import { render } from '../../components/utils/Redux.util';

test('Renders main page', () => {
    const radioWidget = render(<MainPage />);
    expect(radioWidget).toMatchSnapshot();
});
