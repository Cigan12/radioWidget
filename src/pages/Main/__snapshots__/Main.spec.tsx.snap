// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders main page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MainPage"
      >
        <div
          class="Main"
        >
          <div
            class="Header"
          >
            <svg
              class="ArrowLeft"
              height="47.969"
              viewBox="0 0 28 47.969"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              <defs />
              <path
                class="cls-1"
                d="M927.428,154.57a1.9,1.9,0,0,1,0,2.694l-24.159,22.16a1.9,1.9,0,0,1-2.694-2.694l24.158-22.16A1.91,1.91,0,0,1,927.428,154.57Zm-26.856,22a1.9,1.9,0,0,0,0,2.693l24.159,22.161a1.905,1.905,0,0,0,2.694-2.695L903.266,176.57A1.907,1.907,0,0,0,900.572,176.57Z"
                id="interface_-_arrow_left"
                style="fill: #fff; fill-rule: evenodd;"
                transform="translate(-900 -154.031)"
              />
            </svg>
            <h3
              class="Title"
            >
              stations
            </h3>
            <img
              alt="power off"
              class="PowerOff"
              src="powerOff.png"
            />
          </div>
          <div
            class="Content"
          >
            <ul
              class="RadioItems"
            />
          </div>
          <div
            class="Footer"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MainPage"
    >
      <div
        class="Main"
      >
        <div
          class="Header"
        >
          <svg
            class="ArrowLeft"
            height="47.969"
            viewBox="0 0 28 47.969"
            width="28"
            xmlns="http://www.w3.org/2000/svg"
          >
            <defs />
            <path
              class="cls-1"
              d="M927.428,154.57a1.9,1.9,0,0,1,0,2.694l-24.159,22.16a1.9,1.9,0,0,1-2.694-2.694l24.158-22.16A1.91,1.91,0,0,1,927.428,154.57Zm-26.856,22a1.9,1.9,0,0,0,0,2.693l24.159,22.161a1.905,1.905,0,0,0,2.694-2.695L903.266,176.57A1.907,1.907,0,0,0,900.572,176.57Z"
              id="interface_-_arrow_left"
              style="fill: #fff; fill-rule: evenodd;"
              transform="translate(-900 -154.031)"
            />
          </svg>
          <h3
            class="Title"
          >
            stations
          </h3>
          <img
            alt="power off"
            class="PowerOff"
            src="powerOff.png"
          />
        </div>
        <div
          class="Content"
        >
          <ul
            class="RadioItems"
          />
        </div>
        <div
          class="Footer"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
